composer.json

```diff
@@ -59,5 +59,7 @@
                 "Bouncer": "Silber\\Bouncer\\BouncerFacade"
             }
         }
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": false
```

Comment:
Hmmm. Do we really want this?

Maybe we just merge this into master, but remove it before the next Bouncer version is tagged, after Laravel 9.x is released 🤔

Reply:
Whoops! I had added that in for local testing, what that should be is:

```json
{
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

I will push an update with those changes and I would think that would be fine, though we definitely could revert after Laravel is released.

Reply:
So I guess we don't tag it with the minimum stability if dev. We'll tag it right after Laravel is released, but we'll first revert to regular minimum stability.

Reply:
I went ahead and reverted those changes entirely. Tests will be failing until the Laravel 9 release.